biapol_utilities/data/_data.py

```diff
@@ -25,6 +26,188 @@ def blobs():
     return io.imread(os.path.join(data_dir, "blobs.png"))
 
 
+class Hourglass:
```

Comment:
I'm just wondering: Is this class really neccessary? Would it also be possible to put all the code from the __init__ function here into the `hourglass()` function below? I'm asking, because I'm suspecting that this is not a typical use-case for object-oriented programming. A function that produces a dataset sounds to me more like functional programming. Happy to chat about this.

Reply:
Also, as the hourglass code is quite long, I'm suggesting to put it in a _hourglass.py file...

Reply:
I understand that it makes sense to keep hourglass-related functions a bit separated to themselves (thus confining them to a separate class) but I guess moving them to a different file that is then called from _data.py is perfectly feasible as well.